src/config.ts

```diff
@@ -420,3 +423,14 @@ log.info(
   "JWT support token expiration set to %s seconds",
   JWT_SUPPORT_TOKEN_EXPIRATION
 );
+
+export const EMAIL_VALIDATION_PROCESS_TTL: NonNegativeInteger = NonNegativeInteger.decode(
+  process.env.EMAIL_VALIDATION_PROCESS_TTL
```

Comment:
can you add the unit of measure in the name ie. EMAIL_VALIDATION_PROCESS_TTL_MS ?